config/rbac/role.yaml

```diff
@@ -90,6 +99,18 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - mutations.gatekeeper.sh
+  resources:
+  - '*'
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
```

Comment:
This is the relevant portion where the RBAC is no longer consolidated as it was with `controller-gen` v0.16`.